CI/build-mudlet-for-windows.sh

```diff
@@ -135,13 +135,14 @@ if [ "${MSYSTEM}" = "MINGW64" ]; then
   export WITH_OWN_QTKEYCHAIN="NO"
 fi
 
-if [[ "$GITHUB_REPO_TAG" == "false" ]] && [[ "$GITHUB_SCHEDULED_BUILD" == "false" ]]; then
+if [[ "${MUDLET_VERSION_BUILD,,}" == *"-testing"* ]]; then
     # The updater is not helpful in this environment (PR testing build)
     export WITH_UPDATER="NO"
 else
     # Tagged build, this is a release or a PTB build, include the updater
     export WITH_UPDATER="YES"
 fi
+
```

Comment:
\<off-topic>Oh, is this bit still there? Well it is now unused anywhere so it can be removed\</off-topic>